docs/templates/template_reference.md

```diff
@@ -2,17 +2,24 @@
 
 #### accounts_password
 -   Checks if PAM enforces password quality requirements. Checks the
-    configuration in `/etc/pam.d/system-auth` (for RHEL 6 systems) or
-    `/etc/security/pwquality.conf` (on other systems).
+    configuration in `/etc/security/pwquality.conf`.
 
 -   Parameters:
 
-    -   **variable** - PAM `pam_cracklib` (on RHEL 6) or `pam_pwquality`
-        (on other systems) module name, eg. `ucredit`, `ocredit`
+    -   **variable** - PAM `pam_pwquality` password quality
+        requirement, eg. `ucredit`, `ocredit`
 
     -   **operation** - OVAL operation, eg. `less than or equal`
 
--   Languages: OVAL
+    -   **check_zero_boundary** - effective only in OVAL checks, if
+        set to `"true"` it will test if the value respects the a zero
+        boundary operation. The operation is defined by the
+        **zero_boundary_operation** parameter (default value: `"false"`)
```

Comment:
Would it make sense to get rid of `check_zero_boundary` and just use `zero_boundary_operation`?
(Suggestion) And maybe rename it to `zero_comparison_operation`